src/css/main.css

```diff
@@ -197,6 +197,20 @@ h2 {
   margin: 0 0 0.8em 0;
 }
 
+table, td, th {
```

Comment:
Have you checked that this works with other pages in this repo as this will affect all of them using these tags?